### PR TITLE
missing space & wrong tree

### PR DESCRIPTION
**Pull request by bitjimy**
missing space in list, and tree for ((B C) C) and not ((A B) C)

### CODE REVIEW

**Comment by gur111** [comment]
Please merge with master to solve conflicts

**Review by gur111 — COMMENTED** [review]
Conflicts. Yes, I know, it's not your fault. But please fix conflicts and I'll CR and merge

**Comment by bitjimy** [comment]
> Conflicts. Yes, I know, it's not your fault. But please fix conflicts and I'll CR and merge

done.

File: lisp.tex
Comment (gur111) [inline lisp.tex:744]: Add missing dot

File: lisp.tex
Comment (gur111) [inline lisp.tex:770]: Not sure I agree with that fix, @yossigil ?

File: lisp.tex
Comment (gur111) [inline lisp.tex:1080]: @yossigil ?

File: lisp.tex
Comment (gur111) [inline lisp.tex:1122]: @yossigil ?

**Review by gur111 — CHANGES_REQUESTED** [review]
Let's just wait for Yossi's approval on the ones I'm not sure.
Meanwhile fix the dot thing.

**Comment by yossigil** [comment]
I am sorry, but the dev version has converged, and there are still conflicts. Can you please create a new PR with respect to the current dev version.

**Pull request closed without merging**
